cleanup: remove KongIngress/TCPIngress/UDPIngress (#1971)
Kong/kong-operator: CHANGELOG.md

  The default value of the two labels are set to `konghq.com/secret` and
  `konghq.com/configmap`.
  [#1922](https://github.com/Kong/kong-operator/pull/1922)

> - Removed KongIngress, TCPIngress and UDPIngress CRDs and related logics.

Reviewer: ```suggestion
- Removed `KongIngress`, `TCPIngress` and `UDPIngress` CRDs together with their controllers.
```

  deprecated.
  [#1792](https://github.com/Kong/kong-operator/pull/1972)
  <!-- https://github.com/Kong/kong-operator/issues/1944 -->

> - Removed `KongIngress`, `TCPIngress` and `UDPIngress` CRDs together with their controllers.

Reviewer: I believe we should link here to docs explaining the migration for users using these.

Author: Added https://github.com/Kong/kong-operator/pull/1971/commits/00e9547cfe53dfd4594337d37eb1134cdcd3fa10

Reviewer: Nice ! 👍
